Update Getting Started Docs: Calling Analyzers w/ FAKE (#225)
ionide/FSharp.Analyzers.SDK: docs/content/Getting Started Using.md

As you can see, the path to the analyzer DLL files could be tricky to get right across a wide range of setups.  
Luckily, we can use an MSBuild custom target to take care of the path construction.  

> Note: If using MSBuild is not the right solution for you, you can also [call the analyzers from a FAKE build script](#Call-Analyzers-in-Your-FAKE-Build).

Reviewer: I'd just say something simple like:

```suggestion
Note: If you're using Fake you can [call the analyzers from a FAKE build script](#Call-Analyzers-in-Your-FAKE-Build).
```
